src/views/includes/jurisdiction-form.njk

```diff
@@ -0,0 +1,43 @@
+<form class="form" action={{ props.currentUrl }} method="post">
+  <input type="hidden" name="pageType" value={{ props.pageType }}>  
+  <input type="hidden" name="jurisdiction" value={{ props.jurisdiction }}> 
```

Comment:
Not sure whether this is enough to pass the jurisdiction value assuming the selected radio button value would be added but there may be more here.

Reply:
Updated the hidden input to pass value "Scotland" for the name "jurisdiction"